docs/api_reference/template_messages_api/introduction.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+Templates API allows to recover details regarding the template messages used on Callbell.
+
+:::caution
+
+At the moment we **only** support template messages for Whatsapp API (Gupshup) integrations, therefore the only template messages available are the ones approved for WhatsApp.
```

Comment:
```suggestion
At the moment we **only** support template messages for WhatsApp Business API (Gupshup) integrations, which needs to be pre-approved by WhatsApp itself. For further information regarding template approvals please reach out to us.
```

Reply:
I'll add it in a new PR, thanks for the suggestion 👍🏼